### PR TITLE
[PPP] Eliana

### PR DESCRIPTION
**Pull request by eeliana**
- [x] UG extracts
- [x] DG extracts

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 <img src="images/eeliana.png" width="200px">
 
-[[github](http://github.com/eeliana)]
+[[github](http://github.com/eeliana)] [[Portfolio](/team/eeliana.html)]
```
Comment (Xiaoyunnn) [inline docs/AboutUs.md:25]:
```suggestion
[[github](http://github.com/eeliana)] [[Portfolio](team/eeliana.html)]
```
`/team` didnt work for mine

Comment (eeliana) [reply docs/AboutUs.md:25]: Do you think using the .md is better?

Comment (Xiaoyunnn) [reply docs/AboutUs.md:25]: ohh probably. I saw another group using .md. can help me change also?

**Review by Xiaoyunnn — APPROVED** [review]
LGTM

**Pull request merged**